packages/cloudflare/src/api/overrides/tag-cache/tag-cache-filter.ts

```diff
@@ -0,0 +1,58 @@
+import { NextModeTagCache } from "@opennextjs/aws/types/overrides";
+
+interface WithFilterOptions {
+  /**
+   * The original tag cache.
+   * Call to this will receive only the filtered tags.
+   */
+  originalTagCache: NextModeTagCache;
```

Comment:
maybe?
```suggestion
  /**
   * The wrapped tag cache.
   * Call to this will receive only the filtered tags.
   */
  tagCache: NextModeTagCache;
```